android/tangram/jni/platform_android.cpp

```diff
@@ -243,5 +246,27 @@ void setCurrentThreadPriority(int priority) {
     setpriority(PRIO_PROCESS, tid, priority);
 }
 
+void featureSelectionCallback(const std::vector<Tangram::TouchItem>& items) {
+
+    JNIEnv *jniEnv;
```

Comment:
Will this be called from a native thread?

If not, we can do with the cached jniEnv. And we do not need to attach current native thread to the jni environment.